src/ui/autocomplete.ts

```diff
@@ -380,43 +391,41 @@ class AutocompleteUI {
     // add class name to newly highlighted item
     resultItems[index].classList.add(CLASSNAMES.SELECTED_ITEM);
 
+    // set aria active descendant
+    this.inputField.setAttribute('aria-activedescendant', `item-${index}`);
```

Comment:
and if we wanna go with the above suggestion
```suggestion
    this.inputField.setAttribute('aria-activedescendant', `${CLASSNAMES.RESULTS_ITEM}-${index}`);
```